.gitignore

```diff
@@ -46,3 +46,7 @@ rpm/src/
 
```

Comment:
CHANGE_NEXT_RELEASE entry misssing. Not only about implementing the operation (verb + URL) but also about implementing numbers and booleans in attributes (both simple and compound) and metadata in the /v2 API.